src/main/java/io/lettuce/core/protocol/CommandType.java

```diff
@@ -103,6 +103,15 @@ public enum CommandType implements ProtocolKeyword {
 
     XACK, XADD, XAUTOCLAIM, XCLAIM, XDEL, XGROUP, XINFO, XLEN, XPENDING, XRANGE, XREVRANGE, XREAD, XREADGROUP, XTRIM,
 
+    // JSON
+
+    JSON_ARRAPPEND("JSON.ARRAPPEND"), JSON_ARRINDEX("JSON.ARRINDEX"), JSON_ARRINSERT("JSON.ARRINSERT"), JSON_ARRLEN(
+            "JSON.ARRLEN"), JSON_ARRPOP("JSON.ARRPOP"), JSON_ARRTRIM("JSON.ARRAPPEND"), JSON_CLEAR("JSON.CLEAR"), JSON_DEL(
+                    "JSON.DEL"), JSON_GET("JSON.GET"), JSON_MERGE("JSON.MERGE"), JSON_MGET("JSON.MGET"), JSON_MSET(
+                            "JSON.MSET"), JSON_NUMINCRBY("JSON.NUMINCRBY"), JSON_OBJKEYS("JSON.OBJKEYS"), JSON_OBJLEN(
+                                    "JSON.OBJLEN"), JSON_SET("JSON.SET"), JSON_STRAPPEND("JSON.STRAPPEND"), JSON_STRLEN(
+                                            "JSON.STRLEN"), JSON_TOGGLE("JSON.TOGGLE"), JSON_TYPE("JSON.TYPE"),
```

Comment:
That's a weird formatting :P

Reply:
:) I don't particularly like it myself, but it is according to the current formatting rules (autoformated).
We can think about changing them, but certainly not before we get JSON out